plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogMaterializedViewTest.java

```diff
@@ -79,11 +79,17 @@ protected String getSchemaDirectory()
 
     @AfterClass(alwaysRun = true)
     public void cleanup()
+    {
+        cleanUpSchema(schemaName);
+        cleanUpSchema(TESTING_STORAGE_SCHEMA);
```

Comment:
This is going to delete the TESTING_STORAGE_SCHEMA in Glue.
I think we must be using a randomized schema name as the storage schema.

Reply:
I believe it's already the case
```java
protected static final String TESTING_STORAGE_SCHEMA = "testing_storage_schema_" + randomTableSuffix();
```

Reply:
indeed, i think we ended up with code that i am not smart enough to read :) 

any reason schemaName is an instance field  and TESTING_STORAGE_SCHEMA is a constant, if both are initialized in a pretty same manner (a constant randomized at class load time)?

Reply:
I've made storage schema string consistent with the existing `schemaName` field now